src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1263,6 +1453,49 @@ paths:
 
         default:
           $ref: '#/components/responses/defaultResponse'
+
+  /uploads/{id}/item/{itemId}/copyrights:
+    parameters:
```

Comment:
Please add `limit` and `page` parameters from header as done in other paginated endpoints